Services/CO.CDP.OrganisationInformation.Persistence/JoinRequest.cs

```diff
@@ -0,0 +1,21 @@
+using CO.CDP.EntityFrameworkCore.Timestamps;
+using Microsoft.EntityFrameworkCore;
+
+namespace CO.CDP.OrganisationInformation.Persistence;
+
+[Index(nameof(Guid), IsUnique = true)]
+public class JoinRequest : IEntityDate
+{
+    public int Id { get; set; }
+    public required Guid Guid { get; set; }
+    public required Organisation Organisation { get; set; }
+    public required int OrganisationId { get; set; }
+    public required Person Person { get; set; }
```

Comment:
As discussed previously, please mark relational property nullable, so that we don't need to forcefully fetch or set when dealing with entity